mqtt_node:
  ros__parameters:
    robot_id: 1